quadriga/src/main/java/edu/asu/diging/quadriga/service/impl/RepositoryManager.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.diging.quadriga.service.impl;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.json.JSONException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.quadriga.converter.IXmltoObject;
+import edu.asu.diging.quadriga.core.mongo.ICreationEventService;
+import edu.asu.diging.quadriga.domain.elements.Element;
+import edu.asu.diging.quadriga.exceptions.InvalidDataException;
+import edu.asu.diging.quadriga.exceptions.ParserException;
+import edu.asu.diging.quadriga.service.IRepositoryManager;
+
+@Service
+public class RepositoryManager implements IRepositoryManager {
```

Comment:
let's add a deprecation annotation (actually to all classes from the old code that we will remove) to make clear these classes are only here temporary until we have a real api.